monkey/infection_monkey/exploit/powershell.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, host: VictimHost):
 
     def _exploit_host(self):
         try:
-            is_https = self._is_client_using_https()
+            use_ssl = self._is_client_using_https()
```

Comment:
Small note: Ideal case we stick with `is_*` convention for booleans or at least `uses_ssl`

Reply:
Hmm. I was thinking of it from the perspective that we're giving the client instructions, so this is written in the imperative.

> Client: What is thy bidding, my master?
> Exploiter: Execute order 66. Connect to host X. Use ssl.